src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs

```diff
@@ -26,18 +26,24 @@ namespace Tizen.NUI.Scene3D
 {
     /// <summary>
     /// Model is a Class to show 3D mesh objects.
-    /// Model supports to load glTF 2.0 and DLI models for the input format
+    /// Model supports glTF 2.0 and DLI model formats.
     /// and also supports Physically Based Rendering with Image Based Lighting.
```

Comment:
either remove the dot between "formats. and" or change the second line to be a separate sentence, e.g. "Physically Based Rendering with Image Based Lighting is also supported" or "It supports Physically Based Rendering with Image Based Lighting".

Reply:
Thank you I fixed it.